wpilib/wpilib/driverstation.py

```diff
@@ -657,7 +657,7 @@ def _getData(self):
             hal.getJoystickPOVs(stick, self.joystickPOVsCache[stick])
             hal.getJoystickButtons(stick, self.joystickButtonsCache[stick])
 
-        hal.getMatchInfo(self.matchInfoCache)
+        _, self.matchInfoCache = hal.getMatchInfo()
```

Comment:
Have you tried executing this on a rio? Pretty sure it doesn't work.

Reply:
No. No rio anymore. What makes you think it doesn't work?

Reply:
Looks like it only returns a single value: https://github.com/robotpy/robotpy-wpilib/blob/master/hal-base/hal/functions.py#L368

Reply:
https://github.com/robotpy/robotpy-wpilib/blob/master/hal-sim/hal_impl/functions.py#L1063

......o.O